### ISSUE TITLE
Serve clashapi.xyz over HTTPS

### ISSUE THREAD

**Issue by TehSebaXx** [martincarrera/clash-royale-api#83]
If it's possible serve the content over HTTPS for safer use.

**Comment by martincarrera** [comment]
Yes, I will do it in a couple of days. Thanks for opening this issue!

**Comment by TehSebaXx** [comment]
Thank you!

**Comment by martincarrera** [comment]
Just to keep this up, I'm on it and I will try to serve the content over HTTPS in the next couple of days.

Sorry it took so long, I've been really busy lately.

**Comment by chrisbremmer** [comment]
@martincarrera did you ever get around to this?

**Comment by martincarrera** [comment]
@chrisbremmer you can try with [this url](https://clashapi.now.sh/).

I'm not sure if it's production ready but you can start using it.

**Comment by chrisbremmer** [comment]
@martincarrera ah that's actually perfect! I'm just messing around with the api for a bootcamp I'm in at the moment and this saves me having to proxy the request. Heroku didn't like the http :/ Thanks a lot though!

**Comment by efres1990** [comment]
Hi @martincarrera this url https://clashapi.now.sh/ works very well, but is stable? because i am doing a university job and i need with urgency, because heroku and firebase don't works with https.
Thanks and great job.

**Comment by martincarrera** [comment]
@efres1990 it's as stable as I can make it. Take into account that this is a open source project made in my free time and the hosting is being paid by my own money without help, so I cannot get the best hosting in the market.

This said, it is really stable and I try to monitor the server as much as possible, and also if this crashes you can still create a issue.

Also, if you need me to pay special attention at a fixed time for your university job, let me know and I'll make my best.

**Comment by efres1990** [comment]
ok @martincarrera  I understand the situation and thanks. Congratulations for the work, it works really well, if i had some problem i would tell you. Thanks

**Comment by efres1990** [comment]
hi @martincarrera i am having problems with https://clashapi.now.sh/api, the server 
does not respond, please can you check it?

**Comment by martincarrera** [comment]
There is a problem with the provider I'm using. You can check the status [here](https://zeit-status.co/). 

I'm sorry, I'll do all I can but I can't guarantee anything.

**Comment by efres1990** [comment]
:( ok I am pending, thanks

**Comment by efres1990** [comment]
Hi @martincarrera, it seems that it already works. Thank you so much.

**Comment by martincarrera** [comment]
It is, I've just checked that.

Good luck with your college work! Let me know if I can help you with anything :)

**Comment by AmirSavand** [comment]
I also need this @martincarrera when I open with `https://` I get an unsafe warning from google chrome.

**Comment by AmirSavand** [comment]
Error from chrome console:

```
angular.js:110 Mixed Content: The page at 'https://<my-site-here>/' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://www.clashapi.xyz/api/random-deck/'. This request has been blocked; the content must be served over HTTPS.
```

Same for cards.

**Comment by efres1990** [comment]
Hi AmirSavand in my opinion this is because you have in your proyect http and https request you should change this an only use http or https this is not a problem of the api

**Comment by AmirSavand** [comment]
@efres1990 I agree.

**Comment by martincarrera** [comment]
You could try https://clashapi.now.sh in order to have HTTPS support.

I'm still not sure about migrating to this new provider so consider this in 'beta'
